includes/digital-twins-limits.md

```diff
@@ -24,7 +24,7 @@ The table below lists the functional limits of Azure Digital Twins.
 | Models | Number of models within a single Azure Digital Twins instance | 10,000 | Yes |
 | Models | Number of models that can be uploaded in a single API call | 250 | No |
 | Models | Number of items returned in a single page | 100 | No |
-| Query | Number of items returned in a single page | 100 | No |
+| Query | Maximum number of items returned in a single page | 100 | Yes |
```

Comment:
You'll notice that the work "maximum" doesn't appear in any of the others.  The table itself is "Functional limits" and so the limit if the number of items returned... at least that's the wording that we've gone with so far.

Reply:
I find the existing wording confusing because the user can already adjust the limit (downwards) via the `max-items-per-page` header. 

The same is also true of the Models API.

Reply:
It doesn't seem to be unusual to add "Maximum" to limit names (looking at https://docs.microsoft.com/azure/azure-resource-manager/management/azure-subscription-service-limits to view the limit tables for other services). If this seems to more accurately describe this limit, it seems fine to make this change. But also, to @dwalthermsft 's point, we should then look at other similar limits and be consistent with the wording.

Reply:
I'll drop the Maximum for now. We can address any inconsistencies subsequently.